### ISSUE TITLE
Handle double parameters for multipart request

### ISSUE THREAD

**Issue by LivioGama** [s4cha/ws#43]
I send a multipart request with a picture and other parameters. The string ones work but the Double ones don't. I checked the file `WSRequest.swift` in method `func sendMultipartRequest(_ resolve: @escaping (_ result: (Int, [AnyHashable: Any], JSON)) -> Void, reject: @escaping (_ error: Error) -> Void, progress:@escaping (Float) -> Void)`

I add the double case like this : 
```
for (key, value) in self.params {
	if let int = value as ? Int {
		let str = "\(int)"
		if let d = str.data(using: String.Encoding.utf8) {
			formData.append(d, withName: key)
		}
	} else if
	let double = value as ? Double {
		let str = "\(double)"
		if let d = str.data(using: String.Encoding.utf8) {
			formData.append(d, withName: key)
		}
	} else {
		if let str = value as ? String, let data = str.data(using: String.Encoding.utf8) {
			formData.append(data, withName: key)
		}
	}
}
``` 
Would be cool if you could fix it officially

**Comment by maxkonovalov** [comment]
Hello @LivioGama, thanks for the catch! I'd fix this issue with a more generic approach - see #44 
@s4cha what do you think?

**Comment by LivioGama** [comment]
Totally agree for a more generic approach, this is what I was thinking about, thanks !

**Comment by s4cha** [comment]
@maxkonovalov The fix looks good to me ! Thanks for being so reactive as usual :)

**Comment by s4cha** [comment]
Should be solved in 2.0.4  https://github.com/freshOS/ws/releases/tag/2.0.4
Kudos to you guys 👏
